components/org.wso2.carbon.identity.auth.valve/pom.xml

```diff
@@ -218,7 +222,13 @@
                             org.wso2.carbon.identity.core.util.*; version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.core.*; version="${carbon.identity.package.import.version.range}",
                             org.wso2.carbon.identity.auth.service.*;version="${org.wso2.carbon.identity.auth.service.version.range}",
-                            org.apache.catalina.*;version="1.7.0"
+                            org.apache.catalina.*;version="1.7.0",
+                            org.wso2.carbon.identity.oauth.common.exception;version="${org.wso2.carbon.identity.oauth.import.version.range}",
+                            org.wso2.carbon.identity.oauth.common;version="${org.wso2.carbon.identity.oauth.import.version.range}",
+                            org.wso2.carbon.identity.oauth.dao;version="${org.wso2.carbon.identity.oauth.import.version.range}",
+                            org.wso2.carbon.identity.oauth2;version="${org.wso2.carbon.identity.oauth.import.version.range}",
+                            org.wso2.carbon.identity.oauth2.client.authentication;version="${org.wso2.carbon.identity.oauth.import.version.range}",
```

Comment:
Adding  org.wso2.carbon.identity.oauth2 should be enough right?

Reply:
But you have already imported the package  org.wso2.carbon.identity.oauth2 https://github.com/wso2-extensions/identity-carbon-auth-rest/pull/240/files#diff-4b963f95eab8a4f703ce4747c83c47c7cbad093e87ef4f2a2553d7c6d28b3c0eR229

Reply:
It just import oauth package, not the sub packages inside that.
Tried avoid adding `org.wso2.carbon.identity.oauth2.*`. Isn't this what you meant to add?

Reply:
Resolving as per the offline chat